### ISSUE TITLE
Allow to specify multiple patterns delimited by spaces like fzf extended search mode

### ISSUE THREAD

**Issue by mnrce** [neoclide/coc-lists#23]

**Comment by chemzqm** [comment]
I don't use fzf, please explain.

**Comment by mnrce** [comment]
For example, if I type the first pattern: `appts` to fuzzy filter the list, I get this:
![image](https://user-images.githubusercontent.com/13317164/58777547-f593d280-85f8-11e9-9475-86dd69038409.png)

But I want `app.component.ts`, not `app.module.ts`, I can type `<space>` and type the next pattern: `com` to fuzzy filter the remaining item in the list, I get this:
![image](https://user-images.githubusercontent.com/13317164/58777559-09d7cf80-85f9-11e9-8487-dbbfe1ac98b0.png)

So basically fzf allows us to specify multiple patterns to filter the list multiple times.

**Comment by chemzqm** [comment]
It's supported now.
